src/TestFrameworkOrchestrator.ts

```diff
@@ -0,0 +1,48 @@
+import { TestFrameworkFactory, TestFramework } from 'stryker-api/test_framework';
+import { StrykerOptions } from 'stryker-api/core';
+import * as log4js from 'log4js';
+
+const IGNORE_WARNING = 'Set `coverageAnalysis` option explicitly to "off" to ignore this warning.';
```

Comment:
I'll remove the variable. Was used multiple times before, but the new setup is simpler 👍